extensions/spark/kyuubi-spark-authz/src/test/resources/log4j2-test.xml

```diff
@@ -35,7 +35,7 @@
         </File>
     </Appenders>
     <Loggers>
-        <Root level="INFO">
+        <Root level="DEBUG">
```

Comment:
Debug message in CI for authz module can be turned on as most of its logs are at debug level and now it's less noisy.